doc/rados/operations/crush-map-edits.rst

```diff
@@ -644,42 +643,52 @@ There are three types of transformations possible:
-   *default parent* bucket.  In each case, we are careful to preserve
-   the old bucket IDs for the new shadow buckets to prevent data
-   movement.  Any rules with ``take`` steps referencing the old
-   buckets are adjusted.
-
-#. ``--reclassify-bucket <bucket-name> <device-class> <base-bucket>``
-
-   The same command can also be used without a wildcard to map a
-   single bucket.  For example, in the previous example, we want the
-   ``ssd`` bucket to be mapped to the ``default`` bucket.
-
-The final command to convert the map comprising the above fragments would be something like:
-
-.. prompt:: bash $
-
-  ceph osd getcrushmap -o original
-  crushtool -i original --reclassify \
-    --set-subtree-class default hdd \
-    --reclassify-root default hdd \
-    --reclassify-bucket %-ssd ssd default \
-    --reclassify-bucket ssd ssd default \
-    -o adjusted
-
-In order to ensure that the conversion is correct, there is a ``--compare`` command that will test a large sample of inputs against the CRUSH map and check that the same result is output. These inputs are controlled by the same options that apply to the ``--test`` command.  For the above example,:
+   This command reclassifies each bucket that matches a certain
+   pattern. The pattern can be of the form ``%suffix`` or ``prefix%``. For
+   example, in the above example, we would use the pattern
+   ``%-ssd``. For each matched bucket, the remaining portion of the
+   name (corresponding to the ``%`` wildcard) specifies the *base bucket*. All
+   devices in the matched bucket are labeled with the specified
+   device class and then moved to the base bucket. If the base bucket
+   does not exist (for example, ``node12-ssd`` exists but ``node12`` does
+   not), then it is created and linked under the specified
+   *default parent* bucket. In each case, care is taken to preserve
+   the old bucket IDs for the new shadow buckets in order to prevent data
+   movement. Any rules with ``take`` steps that reference the old
+   buckets are adjusted accordingly.
+
+
+   #. ``--reclassify-bucket <bucket-name> <device-class> <base-bucket>``
+
+      The same command can also be used without a wildcard in order to map a
+      single bucket. For example, in the previous example, we want the
+      ``ssd`` bucket to be mapped to the ``default`` bucket.
+
+   #. The final command to convert the map that consists of the above fragments
+      resembles the following:
+
+      .. prompt:: bash $
+
+         ceph osd getcrushmap -o original
+         crushtool -i original --reclassify \
+           --set-subtree-class default hdd \
+           --reclassify-root default hdd \
+           --reclassify-bucket %-ssd ssd default \
+           --reclassify-bucket ssd ssd default \
+           -o adjusted
+
+``--compare`` flag
+------------------
+
+A ``--compare`` flag is available to make sure that the conversion is
+correct.  This flag tests a large sample of inputs against the CRUSH map and
```

Comment:
@zdover23 - A puzzling introduction of double spaces (it's not the product of a sentence beginning on a new line and it's nowhere to be found in the Etherpad edit).

Reply:
Fixed.